stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -80,9 +80,15 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          {{- range $key, $value := .Values.controller.extraEnvs }}
-            - name: {{ $key | upper | replace "." "_" }}
-              value: {{ $value }}
+          {{- if .Values.controller.extraEnvs }}
+            {{- if .Values.controller.inlineExtraEnvs }}
+{{ toYaml .Values.controller.extraEnvs | indent 12 }}
+            {{- else }}
+              {{- range $key, $value := .Values.controller.extraEnvs }}
+              - name: {{ $key | upper | replace "." "_" }}
```

Comment:
This is still indented 2 spaces too many. Same applies elsewhere.

Ex, when rendering this out with inlineExtraEnvs set to false:
```
          env:
            - name: POD_NAME
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name
            - name: POD_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
              - name: FIZZ
                value: buzz
              - name: FOO
                value: bar
          livenessProbe:
```

Reply:
woop 20baa1e